### ISSUE TITLE
training on YFCC100M

### ISSUE THREAD

**Issue by PhazCode** [openai/CLIP#101]
First, thanks for the greak work !
In the paper, you report scores for training a RN50 model on the filtered subset of YFCC100M. (31.3% zero-shot ImageNet accuracy)
Did you use the same parameters as for training on the WIT dataset (in particular, the same batch size) ?
Also, did you train a VIT-B/32 network on that subset ? If yes, did it perform better than the RN50 model ?
Thank you !

**Comment by jongwook** [comment]
We used mostly the same hyperparameters as for the full CLIP models, except a lower weight decay of 0.1 and smaller batch size of 2048. We didn't train a ViT model on YFCC100M subset; our intuition is that it'll perform worse due to the smaller dataset and shorter training.
